### PR TITLE
Add Gas schema - 2

### PR DESCRIPTION
**Pull request by Asura5**
Main flat file table as per Don's DB

image

image

### CODE REVIEW

File: RepairsApi/V2/Infrastructure/Migrations/20220927135406_AddGasSchema2.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "gas_service_data",
+                columns: table => new
+                {
+                    id = table.Column<int>(type: "integer", nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    inspection_ref_number = table.Column<string>(type: "text", nullable: true),
```
Comment (LBHCallumM) [inline RepairsApi/V2/Infrastructure/Migrations/20220927135406_AddGasSchema2.cs:19]: This is probably correct, but InspectionRefNo is defined as a number in the screenshot.

**Comment by LBHCallumM** [comment]
I cannot see the columns `GasProfileDate`, `IsActive` or `LastUpdated`. Just checking that this is intentional.

**Review by LBHCallumM — APPROVED** [review]
Looks good, subject to comments

File: RepairsApi/V2/Infrastructure/Migrations/20220927135406_AddGasSchema2.cs (hunk 1 above)
Comment (Asura5) [reply RepairsApi/V2/Infrastructure/Migrations/20220927135406_AddGasSchema2.cs]: Good spot

**Pull request merged**